api/src/main/java/org/pmiops/workbench/api/ConceptSetsController.java

```diff
@@ -436,4 +363,65 @@ private DbConceptSet getDbConceptSet(
     }
     return conceptSet;
   }
+
+  public static ConceptSet toClientConceptSet(DbConceptSet dbConceptSet) {
+    return new ConceptSet()
+        .etag(Etags.fromVersion(dbConceptSet.getVersion()))
+        .lastModifiedTime(dbConceptSet.getLastModifiedTime().getTime())
+        .creationTime(dbConceptSet.getCreationTime().getTime())
+        .description(dbConceptSet.getDescription())
+        .id(dbConceptSet.getConceptSetId())
+        .name(dbConceptSet.getName())
+        .domain(dbConceptSet.getDomainEnum())
+        .participantCount(dbConceptSet.getParticipantCount())
+        .survey(dbConceptSet.getSurveysEnum())
+        .creator(
+            dbConceptSet.getCreator() != null ? dbConceptSet.getCreator().getUsername() : null);
```

Comment:
nit: You could do
 `Optional.ofNullable(dbConceptSet.getCreator())
.map(Creator::getUsername).orElse(null)`